src/index.js

```diff
@@ -235,6 +235,15 @@ function debugCommand(config, { _chalk = chalk, _process = process, _console = c
           bar.increment(1);
         }
       });
+
+      // Make sure we are setting proper plural forms. This fixes various
+      // issues with gnu gettext. See
+      // https://github.com/mozilla/potools/issues/7 for more details.
+      if (po.headers['Plural-Forms'] === '') {
```

Comment:
This should probably be a falsey check to cover `po.headers['Plural-Forms']`  being undefined too. 

To be fully defensive you could also check `po.headers` too e.g:

```js
if (po.headers && !po.header['Plural-Forms']) {
  // do stuff
}
```